contrib/cloudinary-import.js

```diff
@@ -0,0 +1,71 @@
+const util = require('util')
+const cloudinary = require('cloudinary')
+const fs = require('fs-extra')
+const yaml = require('js-yaml')
+const flatmap = require('lodash.flatmapdeep')
+
+function parseDesignerImages (file) {
+  const obj = yaml.safeLoad(file)
+  return obj.works.map((work) => (
```

Comment:
you may want to ensure that `works` is present

Reply:
Yes - this was a just a quick one-shot import but we should update it to:

1. Fix these potential crashes
2. Do a differential import by looking at the signature in Cloudinary

in case we need to add a lot of images again.